### PR TITLE
Adding a godot 4.0 question to the FAQ.

### PR DESCRIPTION
**Pull request by jacobsky**
This is a starting point, @Bromeon if you have any additional feedback or answers that you would like me to include, please let me know.

### CODE REVIEW

File: src/faq/godot4.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Godot 4.0 Support
+
+## What is the status of Godot 4 Support?
+
+Currently we are still in the planning phase of determining how to support Godot 4.0 and where we will focus our efforts in the future.
+
+## So what is the plan?
+
+While still under development, It's still be developed. This page will be updated as we get more information and planning information.
```
Comment (Bromeon) [inline src/faq/godot4.md]: What do you mean with the first sentence here?
Also, "information and planning information" is duplicating the word.

File: src/faq/godot4.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## What is GDExtension? Why aren't we just upgrading GDNative?
+
+Currently the Godot team has officially announced [GDExtensions](https://godotengine.org/article/introducing-gd-extensions) as the __replacement__ for GDnative. GDNative appears to be no longer supported in Godot 4.0.
+
+## What do we know so far?
+
+Currently we know about the new API is the recently announced [GDExtension](https://godotengine.org/article/introducing-gd-extensions) and we are actively exploring a developing a minimum viable sample with the Code from Godot.
```
Comment (Bromeon) [inline src/faq/godot4.md]: I think the first paragraph could be skipped (it's more or less the same as the answer to the last question). The important parts are written below.

**Review by Bromeon — CHANGES_REQUESTED** [review]
Good idea, thanks!
Few comments below.

File: src/faq/godot4.md (hunk 1 above)
Comment (jacobsky) [reply src/faq/godot4.md]: I'll rewrite this section, I don't actually have an answer to that. :rofl:

Comment (jacobsky) [reply src/faq/godot4.md]: I changed it to: "We don't have any specifics at this time. This page will be updated to reflect the current plan as soon as we have a more concrete plan."

File: src/faq/godot4.md (hunk 2 above)
Comment (jacobsky) [reply src/faq/godot4.md]: Good point. I'll make the changes.

File: src/faq/godot4.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Godot 4.0 Support
+
+## What is the status of Godot 4 Support?
+
+Currently we are still in the planning phase of determining how to support Godot 4.0 and where we will focus our efforts in the future.
+
+## So what is the plan?
+
+We don't have any specifics at this time. This page will be updated to reflect the current plan as soon as we have a more concrete plan.
```
Comment (Bromeon) [inline src/faq/godot4.md]: In the 2nd sentence, I would simply write "This page will be updated as soon as we have a more concrete plan [to share]."

_[ ] means optional, up to you what fits better_

File: src/faq/godot4.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+## What is GDExtension? Why aren't we just upgrading GDNative?
+
+Currently the Godot team has officially announced [GDExtensions](https://godotengine.org/article/introducing-gd-extensions) as the __replacement__ for GDnative. GDNative appears to be no longer supported in Godot 4.0.
+
+## What do we know so far?
+
+Currently as there is limited documentation about how to implement [GDExtension](https://godotengine.org/article/introducing-gd-extensions), most of what we know is being explored via the [godot C++ bindings repo](https://github.com/godotengine/godot-cpp) and determining how to do the following:
```
Comment (Bromeon) [inline src/faq/godot4.md]: "Godot C++ bindings" (uppercase Godot)

**Review by Bromeon — CHANGES_REQUESTED** [review]

File: src/faq/godot4.md (hunk 4 above)
Comment (jacobsky) [reply src/faq/godot4.md]: I really need a lint to catch this. 🤣

**Review by Bromeon — APPROVED** [review]

**Pull request merged**